lib/pwa.js

```diff
@@ -108,6 +122,20 @@ exports.save = function(pwa, callback) {
   });
 };
 
+exports.fetchMetadataDescription = function(url) {
```

Comment:
I'm wondering if this method and it's support libraries should be moved to a different javascript file.